app/controllers/configuration_controller.rb

```diff
@@ -253,6 +254,7 @@ def timeprofile_new
   def timeprofile_edit
     assert_privileges("tp_edit")
     @timeprofile = TimeProfile.find(params[:id])
+    @timeprofile_action = "timeprofile_edit"
```

Comment:
@ZitaNemeckova The code looks good and it works. I just haven't checked the specs. Anyway, I wonder why we need https://github.com/ManageIQ/manageiq-ui-classic/pull/3835/files#diff-679158917bc83d666b7fa919b44817ebR246
 and https://github.com/ManageIQ/manageiq-ui-classic/pull/3835/files#diff-679158917bc83d666b7fa919b44817ebR257 if this is already in `params[;action]` - maybe we could use it instead of `@timeprofile_action`.. but maybe not :D